ibm/service/vpc/data_source_ibm_is_lb_profiles.go

```diff
@@ -46,9 +46,12 @@ func DataSourceIBMISLbProfiles() *schema.Resource {
 										Description: "The type for access mode",
 									},
 									"value": {
-										Type:        schema.TypeString,
+										Type:        schema.TypeList,
 										Computed:    true,
 										Description: "Access modes for this profile",
+										Elem: &schema.Schema{
```

Comment:
this will be a breaking change, can you see if you can rename the field

Reply:
created new field `values`